sys-libs/compiler-rt/compiler-rt-17.0.6-r2.ebuild

```
@@ -117,6 +116,12 @@ src_configure() {
 		-DPython3_EXECUTABLE="${PYTHON}"
 	)
 
+	if use atomic-builtins ; then
+		mycmakeargs+=(
+			-DCOMPILER_RT_EXCLUDE_ATOMIC_BUILTIN=OFF
```

You can just add it to the array, with no explicit conditional, as:
```suggestion
			-DCOMPILER_RT_EXCLUDE_ATOMIC_BUILTIN=$(usex !atomic-builtins)
```

---

Fixed, thanks!